Doc/library/sqlite3.rst

```diff
@@ -521,6 +521,43 @@ Connection Objects
                  f.write('%s\n' % line)
 
 
+   .. method:: backup(filename, *, pages=0, progress=None, name="main")
+
+      This method makes a backup of a SQLite database into the mandatory argument
+      *filename*, even while it's being accessed by other clients, or concurrently by
+      the same connection.
```

Comment:
Maybe mention that backups work for on-disk and in-memory databases like the sqlite3 C api docs mention?